### PR TITLE
add io.github.manisandro.gImageReader

### PR DESCRIPTION
**Pull request by proletarius101**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/manisandro/gImageReader/issues/396#issuecomment-1417559724
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.manisandro.gImageReader.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      - '*.a'
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX='/app'
+      - -DINTERFACE_TYPE=gtk
+      - -DENABLE_VERSIONCHECK=0
+    post-install:
+      # Use the SVG icon instead for clearity
+      - rm -r "${FLATPAK_DEST}/share/icons/hicolor"
+      - install -Dm644 data/icons/gimagereader.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/gimagereader.svg"
+    modules:
```
Comment (hfiguiere) [inline io.github.manisandro.gImageReader.yml:54]: splitting everything to so many is unmaintainable.

File: io.github.manisandro.gImageReader.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      - /share/pkgconfig
+      - /lib/*.la
+      - '*.a'
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX='/app'
+      - -DINTERFACE_TYPE=gtk
+      - -DENABLE_VERSIONCHECK=0
+    post-install:
+      # Use the SVG icon instead for clearity
+      - rm -r "${FLATPAK_DEST}/share/icons/hicolor"
```
Comment (hfiguiere) [inline io.github.manisandro.gImageReader.yml:52]: why removing them?

File: io.github.manisandro.gImageReader.yml (hunk 1 above)
Comment (hfiguiere) [reply io.github.manisandro.gImageReader.yml:54]: no it is not. been there, done that. "copied from other" is a fallacy.

File: io.github.manisandro.gImageReader.yml (hunk 2 above)
Comment (hfiguiere) [reply io.github.manisandro.gImageReader.yml:52]: so I'll rephrase:

```suggestion
```

File: io.github.manisandro.gImageReader.yml (hunk 1 above)
Comment (hfiguiere) [reply io.github.manisandro.gImageReader.yml:53]: yes and build in order (ie the top level module list).

valid exceptions are shared modules and auto-generated manifests.

Comment (hfiguiere) [reply io.github.manisandro.gImageReader.yml:53]: right now it doesn't track either.

**Comment by barthalion** [comment]
@proletarius101 Noticed post-merge that the desktop icon is not being used by the desktop launcher and the about popup.

image

Not critical enough to cancel the merge, but it should be looked at.

**Comment by barthalion** [comment]
Although that might be problem on my end. If yes, ignore me.

**Pull request closed without merging**